code/game/verbs/ooc.dm

```diff
@@ -209,6 +209,9 @@ GLOBAL_VAR_INIT(admin_ooc_colour, "#b82e00")
 
 	log_looc(msg, src)
 	mob.create_log(LOOC_LOG, msg)
+	for(var/mob/M in get_mobs_in_view(7, mob))
+		if((M.client?.prefs.toggles2 & PREFTOGGLE_2_RUNECHAT) && M.can_hear())
```

Comment:
If it's LOOC, shoudn't it use `viewers()` or something instead of `can_hear()`? Deaf people should see LOOC

Reply:
You're right, that's a copy paste mistake on my part